service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProvider.java

```diff
@@ -14,34 +18,25 @@
 @SuppressWarnings("VariableDeclarationUsageDistance")
 public abstract class AbstractEmailContentProvider {
 
-    private static final String ORDER_KEY = "orders";
-    private static final String DIRECTIONS_KEY = "directionsAndInterim";
-
     private final String uiBaseUrl;
 
     protected AbstractEmailContentProvider(String uiBaseUrl) {
         this.uiBaseUrl = uiBaseUrl;
     }
 
-    protected ImmutableMap.Builder<String, String> getCasePersonalisationBuilder(CaseDetails caseDetails) {
+    protected ImmutableMap.Builder<String, Object> getCasePersonalisationBuilder(CaseDetails caseDetails) {
         String dataPresent = "Yes";
         String fullStop = "No";
         String timeFramePresent = "No";
 
         Map orders =
-            Optional.ofNullable((Map) caseDetails.getData().get(ORDER_KEY)).orElse(ImmutableMap.builder().build());
+            Optional.ofNullable((Map) caseDetails.getData().get("orders")).orElse(ImmutableMap.builder().build());
+        ImmutableSet.Builder<String> ordersAndDirectionsBuilder = ImmutableSet.builder();
 
-        List orderType = (List) Optional.ofNullable(orders.get("orderType")).orElse(ImmutableList.builder().build());
-        String directions = (String) Optional.ofNullable(orders.get(DIRECTIONS_KEY)).orElse("");
-
-        ImmutableMap.Builder<String, String> orderTypeArray = ImmutableMap.builder();
-        for (int i = 0; i < 5; i++) {
-            if (i < orderType.size()) {
-                orderTypeArray.put(ORDER_KEY + i, "^" + orderType.get(i));
-            } else {
-                orderTypeArray.put(ORDER_KEY + i, "");
-            }
-        }
+        boolean isDataPresent = buildOrders(orders, ordersAndDirectionsBuilder);
```

Comment:
Rename `isDataPresent` variable, it is not a method, it should be `dataPresent`.